docs/01-start-here/06-testing-tips.md

```diff
@@ -0,0 +1,157 @@
+# Testing tips
+
+### Get credentials from janus
+Before you do anything else on your local frontend build, navigate to the janus page
+and follow the steps to get your credentials for Frontend and for CMS-Fronts.
+This will save you grief later on.
+
```

Comment:
Before getting on to running sbt it might be worth mentioning running make install/make compile for the JavaScript?